mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -0,0 +1,28 @@
+use crate::bridge::NestedBridge;
+use crate::settings::Settings;
+use tracing::info;
+
+#[derive(Default)]
+pub struct BridgeController {
+    nested_bridge: Option<NestedBridge>,
+}
+
+impl BridgeController {
+    pub fn new() -> Self {
```

Comment:
nit: not necessary, IMO, if you already have `default()`.

Reply:
we might need to pass the config file path to the broker controller, not sure how it will look like so will leave the ctor for now